featherpad/fpwin.cpp

```diff
@@ -695,7 +695,7 @@ void FPwin::applyConfigOnStarting()
         config.setReservedShortcuts (reserved);
         config.readShortcuts();
     }
-
+    
```

Comment:
Would you please remove this redundant change and also put the OS/2-specific file `featherpad.rc` inside the `data` directory?